src/drivers/mct/cime_config/config_component.xml

```diff
@@ -1804,6 +1804,17 @@
     </desc>
   </entry>
 
+  <entry id="NODE_FAIL_REGEX">
+    <type>char</type>
+    <default_value>UNSET</default_value>
```

Comment:
I've been trying to move away from UNSET as a default <default_value\> will set it to None.